README.md

```diff
@@ -145,3 +175,30 @@ pip install -e . --no-build-isolation
 See the `sklearn_numba_dpex/tests` folder for example usage.
 
 TODO: write some doc here instead.
+
+### Running the benchmarks
+
+The commands:
+
+```
+cd benckmark
+python ./kmeans.py
+```
+
+will run a benchmark for different k-means implementations and print a short summary of the performances.
+
+Some parameters in the `__main__` section of the file `./benchmark/kmeans.py` are exposed for quick edition (`n_clusters`, `max_iter`, `skip_slow`, ...).
+
+### Notes about the preferred floating point precision
+
+In many machine learning applications, operations using single-precision (float32) floating point data are regarded as faster, accurate enough and more suitable for GPU compute. Besides, most GPUs used in machine learning projects are significantly faster with float32 than with double-precision (float64) floating point data.
```

Comment:
Probably I can get a "Captain Obvious" achievement for this suggestion so let's try.
```suggestion
In many machine learning applications, operations using single-precision (float32) floating point data require twice as less memory that double-precision (float64), are regarded as faster, accurate enough and more suitable for GPU compute. Besides, most GPUs used in machine learning projects are significantly faster with float32 than with double-precision (float64) floating point data.
```